package.json

```diff
@@ -93,7 +93,6 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.10.5",
     "@octokit/core": "^3.1.2",
-    "browserslist-config-terra": "^1.2.0",
```

Comment:
not sure how this guy got in here.

Reply:
This is used with the stylelint compat plugin